### DISCUSSION TITLE
Não consigo lançar requisições para a controller

### DISCUSSION THREAD

**Discussion by elvisreisnf** [balta-io/forum#159] [Tira Dúvidas] [answered]
Olá, estou no curso de ASP.Net e tentando fazer uma requisição para a Controller da erro 404, sendo que a requisição para o Program.cs funciona. Aparentemente, não vejo motivos para tal erro. (Posso estar ficando doido, por favor, me ajude a achar).

![image](https://user-images.githubusercontent.com/72757269/207166455-f5fdc088-242e-48df-8dc7-e3655d86113d.png)

![image](https://user-images.githubusercontent.com/72757269/207166621-802e6975-3d89-408e-b98c-ede0c0b85a32.png)

**Comment by elvisreisnf** [comment]
Seguem os prints com as respostas:

![image](https://user-images.githubusercontent.com/72757269/207166835-2297f310-b001-4f5c-a18a-5fa8f568543a.png)

![image](https://user-images.githubusercontent.com/72757269/207166923-002dd5ec-36fe-4898-9955-e0d0d32b4481.png)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @elvisreisnf, como vai? No caso, você está usando Minimal APIs, neste formato os controllers não são mapeados automaticamente.

Você precisa utilizar o `app.UseControllers();` e `app.MapControllers` para que eles sejam reconhecidos.

**Reply by elvisreisnf** [reply]
Mas no curso não utiliza isso, então achei que não precisaria de nada mais.

**Reply by elvisreisnf** [reply]
Obrigado pela resposta! Vou desenvolver!!

**Reply by andrebaltieri** [reply]
Ahhh desculpe... precisa sim!! Isso aí, bora lá 🚀
